.changeset/witty-suns-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sebgroup/green-core": minor
+---
+
+Refactor Icon
```

Comment:
Here it would be good with a bit more information of the new feature, for example "added new icon set from Central" and that icons are used as components

Reply:
Added a recap of changes on the file ✌️